Makefile

```diff
@@ -0,0 +1,135 @@
+# Shortcuts for various dev tasks. Based on makefile from pydantic
+.DEFAULT_GOAL := all
+isort = isort -rc src tests
+black = black src tests
+
+VERSION := $(shell python3 setup.py -V)
+RPMDIR := "$(shell pwd)/rpms"
+
+ifndef $(RELEASE)
+RELEASE := dev
+endif
+
+ifeq ($(BUILDID),)
+TIMESTAMP := $(shell date --utc +%Y%m%d%H%M)
+   ifeq ("$(RELEASE)","dev")
+BUILDID := .dev$(TIMESTAMP)
+BUILDID_EGG := .dev$(TIMESTAMP)
+   endif
+   ifeq ("$(RELEASE)","next")
+BUILDID := .next$(TIMESTAMP)
+BUILDID_EGG := rc$(TIMESTAMP)
+   endif
+endif
+
+ifeq ($(RELEASE), dev)
+ISO_REPO := inmanta-service-orchestrator-3-dev
+REPOMANAGER_REPO := inmanta-service-orchestrator-dev/3
+endif
+ifeq ($(RELEASE), next)
+ISO_REPO := inmanta-service-orchestrator-3-next
+REPOMANAGER_REPO := inmanta-service-orchestrator-next/3
+endif
+ifeq ($(RELEASE), stable)
+ISO_REPO := inmanta-service-orchestrator-3
+REPOMANAGER_REPO := inmanta-service-orchestrator/3
+endif
+
+.PHONY: install
+install:
+	pip install -U setuptools pip
+	pip install -U -r requirements.txt
+	pip install -e .
+
+.PHONY: format
+format:
+	$(isort)
+	$(black)
+
+.PHONY: build-pytest-inmanta-extensions
+build-pytest-inmanta-extensions:
```

Comment:
Is this target required for the `inmanta` repo?

Reply:
no, probably not

Reply:
fixed